CHANGELOG.md

```diff
@@ -9,8 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Add usage example for Ray Train, see [docs/torch/ray_usage.rst](https://github.com/microsoft/DeepGNN/tree/main/docs/torch/ray_usage.rst).
 
+- Add documentation for Ray Data usage, see [tutorial](https://github.com/microsoft/DeepGNN/tree/main/docs/graph_engine/dataset.rst) and [example](https://github.com/microsoft/DeepGNN/tree/main/docs/graph_engine/ray_usage_advanced.rst)
+
 - Breaking. Rename get_feature_type -> get_python_type.
 
+### Fixed
+- ~1Mb leak when reset is not called before deleting python graph engine object.
```

Comment:
nit: `Implement `__del__` method to release C++ client and server. Important for ray actors, because they create numerous clients during training.`